datafusion-physical-expr/Cargo.toml

```diff
@@ -41,3 +41,4 @@ arrow = { version = "9.0.0", features = ["prettyprint"] }
 paste = "^1.0"
 ahash = { version = "0.7", default-features = false }
 ordered-float = "2.10"
+roaring = "0.8.1"
```

Comment:
What would you think about making this an optional dependency (much like crypto expressions, etc)  as defined on master?

This would let anyone who wants this feature be able to use it, but would not require it for anyone who did not?

https://github.com/apache/arrow-datafusion/blob/7eb3bd8/datafusion-physical-expr/Cargo.toml#L35-L39